src/test/java/com/google/songgame/TitleFormatterTest.java

```diff
@@ -0,0 +1,128 @@
+package com.google.songgame.data;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/*
+  https://music.youtube.com/playlist?list=RDCLAK5uy_kmPRjHDECIcuVwnKsx2Ng7fyNgFKWNJFs
+
+*/
+
+@RunWith(JUnit4.class)
+public final class TitleFormatterTest {
+
+  @Test
+  public void simpleCapitalization() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "Marvins Room";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "marvins room");
+
+    testString = "BLACK PARADE";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "black parade");
+  }
+
+  @Test
+  public void removeOneGroup() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "Sativa (ft. Swae Lee)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "sativa");
+
+    testString = "Get You (feat. Kali Uchis)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "get you");
+
+    testString = "Promises (feat. Namiko, Miyagi & Miyagi & Namiko)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "promises");
+
+    testString = "Tattoo (Remix)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "tattoo");
+  }
+
+  @Test
+  public void removeMultipleGroups() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "Savage (Remix) (feat. Beyoncé)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "savage");
+
+    testString = "Life is Good (Official Music Video) (feat. Drake)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "life is good");
+  }
+
+  @Test
+  public void removeMultipleGroupsSquareAndParen() {
+    TitleFormatter titleFormatter = new TitleFormatter();
+
+    String testString = "WHATS POPPIN [Remix] (feat. DaBaby, Tory Lanez & Lil Wayne)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "whats poppin");
+
+    testString = "Djadja [Remix] (feat. Afro B)";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "djadja");
+
+    testString = "Roses (Imanbek Remix) [Latino Gang]";
+    Assert.assertEquals(titleFormatter.formatVideoTitle(testString), "roses");
+  }
+
+  @Test
+  public void removeValidGroups() {
```

Comment:
I'm a little confused by this test. What are valid groups and why would we remove them? I think maybe this test is showing that we *don't* remove valid groups.

In any case, I think you should either make the method name more explicit, or add a javadoc explaining what this is testing